roles/common/molecule/default/group_vars/all.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
What is the difference between setting variables in the defaults for hte common role and setting them here?